contracts/land/LANDStorage.sol

```diff
@@ -10,9 +10,11 @@ contract LANDStorage {
   uint256 constant clearHigh = 0x00000000000000000000000000000000ffffffffffffffffffffffffffffffff;
   uint256 constant factor = 0x100000000000000000000000000000000;
 
-  mapping (address => bool) public authorizedDeploy;
+  mapping (address => bool) internal _deprecated1;
```

Comment:
Maybe we can call it more specific:
`_deprecated_authorizedDeploy`
`_deprecated_authorizedDeploy_11_01_2018` (with a date)

Reply:
I think it's a good idea, will take the first option. We can get when it was changed from the commit log.